.travis.yml

```diff
@@ -1,11 +1,22 @@
 ---
 language: node_js
+node_js:
+  - "4"
+  - "6"
+  - "stable"
 
 sudo: false
 
 cache:
-  directories:
-    - node_modules
+  yarn: true
+
+before_script:
+  # ensure we are testing against latest versions of all deps when using Yarn
+  - yarn upgrade
+
+branches:
+  only:
+    - master
```

Comment:
I can't have my own fork (transfered from `rwblue` to `ember-cli`) so I added this because I don't want to kick off two CI jobs for my PR branch pushes.